Ports/harfbuzz/patches/0002-Build-against-freetype.patch

```diff
@@ -0,0 +1,26 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Dominika Liberda <ja@sdomi.pl>
+Date: Tue, 13 Jun 2023 23:09:19 +0200
+Subject: [PATCH] Build against freetype
+
+---
+ CMakeLists.txt | 6 ++----
+ 1 file changed, 2 insertions(+), 4 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 9a1d9032fcde002c27ab5b06c9665c1ca843d13a..c654284a3e6b9511b0433b8c0d25fccb57df30fb 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -37,10 +37,8 @@ option(HB_HAVE_FREETYPE "Enable freetype interop helpers" OFF)
+ option(HB_HAVE_GRAPHITE2 "Enable Graphite2 complementary shaper" OFF)
+ option(HB_HAVE_GLIB "Enable glib unicode functions" OFF)
+ option(HB_HAVE_ICU "Enable icu unicode functions" OFF)
+-if (TARGET freetype)
+-  set (HB_HAVE_FREETYPE ON)
+-  add_definitions(-DHAVE_FREETYPE=1)
+-endif ()
++set (HB_HAVE_FREETYPE ON)
++add_definitions(-DHAVE_FREETYPE=1)
```

Comment:
Hm, isn't it possible to enable the target instead of modifying the Cmake source?

Reply:
adding a CMake flag didn't seem to do anything. CMake even complained that it's unused by the project.

Reply:
IIRC we only tried `-Dwhatever`, might be worth trying `--target freetype` as well

Reply:
as in

```diff
diff --git a/Ports/harfbuzz/package.sh b/Ports/harfbuzz/package.sh
index b4681f7039..6351211f05 100755
--- a/Ports/harfbuzz/package.sh
+++ b/Ports/harfbuzz/package.sh
@@ -5,7 +5,13 @@ files="https://github.com/harfbuzz/harfbuzz/releases/download/${version}/harfbuz
 useconfigure='true'
 auth_type='sha256'
 depends=("freetype" "libicu")
-configopts=("-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt" "-DHB_HAVE_FREETYPE=ON" "-DHB_HAVE_ICU=ON")
+configopts=(
+    "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
+    "-DHB_HAVE_FREETYPE=ON"
+    "-DHB_HAVE_ICU=ON"
+    # Sets HB_HAVE_FREETYPE=ON and -DHAVE_FREETYPE=1
+    "--target freetype"
+)
 
 configure() {
     run mkdir -p build
```

Reply:
`--target freetype` doesn't do anything, and I can't really find documentation on how `--target` works in relation to that if that i'm patching out. I'm kind of at a loss here :(

Reply:
okies, we'll just patch it then :)